frontend/javascripts/oxalis/model/bucket_data_handling/layer_rendering_manager.ts

```diff
@@ -175,58 +186,56 @@ export default class LayerRenderingManager {
     return this.textureBucketManager.getTextures();
   }
 
-  // Returns the new anchorPoints if they are new
-  updateDataTextures(position: Vector3, logZoomStep: number): Vector4 | null | undefined {
+  getLookUpCuckooTable() {
+    return getLookUpCuckooTable();
+  }
+
+  updateDataTextures(position: Vector3, logZoomStep: number): void {
     const state = Store.getState();
     const { dataset, datasetConfiguration } = state;
-    const isAnchorPointNew = this.maybeUpdateAnchorPoint(position, logZoomStep);
     const layer = getLayerByName(dataset, this.name);
     const resolutionInfo = getResolutionInfo(layer.resolutions);
-    const datasetResolutionInfo = getDatasetResolutionInfo(dataset);
     const maximumResolutionIndex = resolutionInfo.getHighestResolutionIndex();
 
     if (logZoomStep > maximumResolutionIndex) {
       // Don't render anything if the zoomStep is too high
-      this.textureBucketManager.setActiveBuckets([], this.cachedAnchorPoint, isAnchorPointNew);
-      return this.cachedAnchorPoint;
+      this.textureBucketManager.setActiveBuckets([]);
+      return;
     }
 
     const resolutions = getResolutions(dataset);
-    const subBucketLocality = getSubBucketLocality(
-      position,
-      resolutionInfo.getResolutionByIndexWithFallback(logZoomStep, datasetResolutionInfo),
+
+    const layerMatrix = invertAndTranspose(layer.transformMatrix || Identity4x4);
+
+    const matrix = M4x4.scale1(
+      state.flycam.zoomStep,
+      M4x4.mul(layerMatrix, state.flycam.currentMatrix),
     );
-    const areas = getAreasFromState(state);
-    const matrix = getZoomedMatrix(state.flycam);
+
     const { viewMode } = state.temporaryConfiguration;
-    const isArbitrary = constants.MODES_ARBITRARY.includes(viewMode);
     const { sphericalCapRadius } = state.userConfiguration;
     const isVisible = isLayerVisible(dataset, this.name, datasetConfiguration, viewMode);
+    const rects = getViewportRects(Store.getState());
```

Comment:
I think, the "areas" are a concept which are a bit outdated now. They basically contain top, left, right, bottom in bucket coordinates. This means that the areas didn't necessarily change when one moved only by one voxel (since the same buckets were at the border of the viewport). With the transformation of layers, this doesn't really make much sense anymore, though. The oblique bucket picker needs the exact extent of the rendered viewport to calculate its things (one could probably use a similar optimization technique, though, but I don't feel like this would get us much).

The areas are only used by the prefetching saga now. This could use some overhauling, too, but obviously it's out of scope for this PR.